src/routes/List.jsx

```diff
@@ -0,0 +1,55 @@
+import FetchAPI from "../components/FetchAPI";
+import Loading from "../components/Loading";
+import Error from "../components/Error";
+
+export default function List() {
+  const { countries, loading, error } = FetchAPI();
+
+  if (loading) {
+    return <Loading />;
+  }
+
+  if (error) {
+    return <Error message={error.message} />;
+  }
+
+  return (
+    <div className="p-4 bg-gradient-to-t from-slate-400 to-slate-600">
+      <h2 className="mb-4 text-3xl font-bold text-center text-white sm:text-left">
+        List of Countries from South America
+      </h2>
+      <div className="grid grid-cols-1 gap-6 sm:grid-cols-3 lg:grid-cols-4">
+        {countries.map((country) => (
+          <div key={country.id} className="p-4 bg-white rounded-lg">
+            <img
+              src={country.flag_png ? country.flag_png : "N/A"}
+              alt={
+                country.flag_alt
+                  ? country.flag_alt
+                  : `A picture of the country ${countries.name}`
+              }
```

Comment:
It may have been a good idea to  have a div or other container around the image only and set it to a certain size. This would allow the text to line up on all cards